README.md

```diff
@@ -1,79 +1,81 @@
+   
 ## Inputs
-| Parameter | Is Required | Default | Description           |
-| --------- | ----------- | ------- | --------------------- |
-| `input-1` | true        |         | Description goes here |
-| `input-2` | false       |         | Description goes here |
+| Parameter            | Is Required | Description                                                                                                                                      |
+| -------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `github-token`       | true        | A token with permission to create and delete releases.  Generally secrets.GITHUB_TOKEN.                                                          |
+| `upload-url`         | true        | The URL for uploading assets to the release.  Can be taken from the outputs of an api call to create a release like in [im-open/create-release]. |
+| `asset-path`         | true        | The path to the asset you want to upload.                                                                                                        |
+| `asset-name`         | true        | The name of the asset you want to upload.                                                                                                        |
+| `asset-content-type` | true        | The content-type of the asset you want to upload. See the [supported Media Types].                                                               |
 
 ## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+| Output                       | Description                                                       |
+| ---------------------------- | ----------------------------------------------------------------- |
+| `asset-browser-download-url` | URL users can navigate to in order to download the uploaded asset |
 
 ## Usage Examples
 
 ```yml
-# TODO: Fill in the correct usage
+on: 
+  pull_request:
+    types: [opened, reopened, synchronize]
+
 jobs:
-  job1:
-    runs-on: ubuntu-20.04
+  create-release:
+    runs-on: ubuntu-latest
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.asset-upload-url }}
+
     steps:
       - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+
+      - name: Calculate next version
+        id: version
+        uses: im-open/git-version-lite@v1.0.0
+        with:
+          calculate-prerelease-version: true
+          branch-name: ${{ github.head_ref }}
 
-      - name: Add Step Here
-        uses: im-open/this-repo@v1.0.0
+      - name: Create Pre-release
+        id: create_release
+        uses: im-open/create-release@v1.0.0
         with:
-          input-1: 'abc'
-          input-2: '123
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          tag-name: ${{ steps.version.outputs.VERSION }}
```

Comment:
This is just an example so it doesn't matter much, but since the `version` step is calculating a pre-release and the name of this step is `Create Pre-release`, should the `prerelease` property be set?